packages/example/src/Features/index.tsx

```diff
@@ -1,9 +1,22 @@
+import {FC, useEffect, useState} from 'react';
 import {Video} from 'remotion';
 
-const Features: React.FC = () => {
+const Features: FC = () => {
+	const [canPlay, setCanPlay] = useState<boolean>();
 	const tray = require('./tray.webm');
 	const watermelon = require('./watermelon.webm');
 	const textstickers = require('./textstickers.webm');
+
+	useEffect(() => {
+		const videoEl = document.createElement('video');
```

Comment:
Out of curiosity, which browser do you use that doesn't support playing WebM videos?

Reply:
Safari doesn't play it and throws errors.